mortar/src/main/java/mortar/MortarScope.java

```diff
@@ -204,6 +204,11 @@ void assertNotDead() {
       this.parent = parent;
     }
 
+    /**
+     * Makes this service available via the new scope's {@link MortarScope#findService}
+     * method. If the service implements {@link Scoped} it will be registered with
+     * the new scope.
+     */
     public Builder withService(String serviceName, Object service) {
```

Comment:
discussed offline, but for the record:  now tend to agree with PY, a second method with a Scoped param is probably better than the instanceof check.  The method w/ Object could do an instanceof check to guard against accidental usage of the wrong method.